provisioning/miniprov/hare_mp/dhall/gencdf.dhall

```diff
@@ -9,7 +9,8 @@ let NodeInfo =
       , data_iface : Text
       , data_iface_type : Optional T.Protocol
       , io_disks : List Text
-      , meta_data : Text
+      , meta_data1 : Text
```

Comment:
@SwapnilGaonkar7 Are you sure that 2 metadata items will suffice? And vice versa, do we think that there will always not less than 2 metadata volumes?

I just think that it makes more sense to have list of metadata volumes. But probably I miss something, please advice.

Reply:
Please find below discussion,
**meta_data for first ioservice: /dev/vg_{nodename}_md1/lv_raw_md1
meta_data for secondioservice: /dev/vg_{nodename}_md2/lv_raw_md2
for ex:
/dev/vg_srvnode-1_md1/lv_raw_md1
/dev/vg_srvnode-1_md2/lv_raw_md2**
So, currently number of metadata volumes are equal to number of IO services in particular node and number of IO services are fixed to 2 so 2 metadata items should suffice.

@knekrasov Let me know your input
.

Reply:
For me that message sounds like something intermittent that can change again quite soon.

My point is that it is simpler and more flexible if from Dhall types level we use list. Then once the number of metadata volumes changes, we'll not need to add meta_data3 or meta_data9.